Fixes issue #557

amarzavery opened the pull request.

amarzavery commented:
@wbreza - Please review the PR

wbreza commented on AutoRest/Generators/NodeJS/NodeJS/TemplateModels/ModelTemplateModel.cs:
```diff
@@ -171,11 +171,31 @@ public bool ContainsPropertiesInSequenceType()
         {
-            var sample = ComposedProperties.FirstOrDefault(p => 
-                p.Type is CompositeType || p.Type is SequenceType && (p.Type as SequenceType).ElementType is CompositeType);
+            var sample = ComposedProperties.FirstOrDefault(p => ContainsCompositeType(p.Type));
```
Code looks fine - Do we have any unit tests to validate the change?

amarzavery commented:
@webreza- I have added a simple test. You will see other nodejs file generated because we updated the version of code generator to 0.14.0 .you can take a look at complexModelClient folder
